src/flick/suggestion/tests.py

```diff
@@ -8,7 +8,7 @@
 from show.models import Show
```

Comment:
idk why github is like this lol... maybe cuz i deleted test_show_rating.py

Reply:
looool gotcha